### DISCUSSION TITLE
[깃헙 활용]이슈 및 브랜치 생성 방법

### DISCUSSION THREAD

**Discussion by Chominsu98** [readysetG0/readysetG0-frontend#22] [지식자료공유]
## 커밋할 시 이슈번호를 사용하는 이유
[좋은 점](https://velog.io/@junh0328/%ED%98%91%EC%97%85%EC%9D%84-%EC%9C%84%ED%95%9C-%EA%B9%83%ED%97%88%EB%B8%8C-%EC%9D%B4%EC%8A%88-%EC%9E%91%EC%84%B1%ED%95%98%EA%B8%B0)

## 이슈에서 바로 브랜치 생성법 예시
![image](https://github.com/user-attachments/assets/01ac092b-d6e0-430f-9828-1ed7b355553a)

## 이슈생성 및 브랜치 생성 컨벤션
![image](https://github.com/user-attachments/assets/cd42bd22-cfcf-44f2-b622-018456484d72)

[우리도 해당 기준으로 이슈생성 및 브랜치 생성 진행](https://velog.io/@rmaomina/pre-project-github-features-convensions)
대신 우리는 브랜치 전략은 뒤에 기능구현까지 적어주자
ex) [FEAT]#1/오픈스트리트맵 연동

**Comment by Oneul1213** [comment]
브랜치 네이밍 컨벤션 수정
- 이유 : 제일 앞 [ 인식 안 됨

{타입}{이슈번호}/{내용}
ex) FEAT#1/오픈스트리트맵연동
- 띄어쓰기 사용 시 - 사용 (그렇게 자동 생성됨)
ex) FEAT#1/웹뷰-띄우기
